plugins/extractors/bigquery/bigquery.go

```diff
@@ -432,15 +435,25 @@ func (e *Extractor) buildAsset(ctx context.Context, t *bigquery.Table, md *bigqu
 		e.logger.Warn("error creating Any struct", "error", err)
 	}
 
-	return &v1beta2.Asset{
+	asset := &v1beta2.Asset{
 		Urn:         tableURN,
 		Name:        t.TableID,
 		Type:        "table",
 		Description: md.Description,
 		Service:     "bigquery",
 		Data:        table,
 		Labels:      md.Labels,
-	}, nil
+	}
+
+	if e.config.BuildViewLineage && (md.Type == bigquery.ViewTable || md.Type == bigquery.MaterializedView) {
+		query := getViewQuery(md)
+		upstreamResources := getUpstreamResources(query)
+		asset.Lineage = &v1beta2.Lineage{
+			Upstreams: upstreamResources,
+		}
+	}
+
+	return asset, nil
```

Comment:
would it possible we cover this in test?

Reply:
As discussed, this is not possible through current test setup as BQ emulator doesn't support VIEW